### DISCUSSION TITLE
Turtle in flet

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [flet-dev/flet#1075] [Q&A] [answered]
can we use turtle module in flet

**Comment by iron3oxide** [comment, marked as answer]
That depends on how you want to use it. Since turtle uses `tkinter` as the underlying GUI framework, it could at best be embedded into a flet app (don't know if that's possible at the moment) or opened in a separate window. If you want to use turtle in a non-external way from within flet, you'll have to write a turtle library for that yourself. Could be a fun project to learn flet!

**Reply by saurabhwadekar** [reply]
thank you brother.
i will try it
